rptest: disable kerberos and tls_crc32 test

ivotron opened the pull request:
These tests are failing on newer CI infra and are blocking upgrades of underlying pools.

https://redpandadata.atlassian.net/browse/CORE-13761

## Backports Required

<!-- Checking at least one of the checkboxes is REQUIRED if this PR is not a backport. -->

- [x] none - not a bug fix
- [ ] none - this is a backport
- [ ] none - issue does not exist in previous branches
- [ ] none - papercut/not impactful enough to backport
- [ ] v25.2.x
- [ ] v25.1.x
- [ ] v24.3.x

## Release Notes

* none

dotnwat commented on tests/rptest/tests/tls_metrics_test.py:
```diff
@@ -314,39 +314,39 @@ def test_expiry_reload(self):
-        reloaded = check_crc()
-
-        assert original != reloaded, f"Checksums unexpectedly equal"
+#    @cluster(num_nodes=3)
```
- does @ignore not work?
- also, do we have a way to surface disabled/ignored tests? i have a scary feeling that we disable tests for various ephemeral reasons and never think about them again 😨 lol

ivotron replied:
right. i'll use ignore instead

dotnwat replied:
i wonder what it's like being https://github.com/ignore

rockwotj replied:
I'm fine with ignoring these, but do they still work in CDT? Your changes for spot instances are only for CI right? We could keep coverage in CDT if that's not changing

ivotron replied:
yeah. i switched to using the suite YAML file instead in the latest force-push. in that way this is executed in CDT and skipped on PRs/merges. LMK if that works

ivotron committed:
rptest: remove commented test and use suite

Use the suite YAML file to skip the test instead of commenting it out in
code.

Signed-off-by: Ivo Jimenez <ivo@redpanda.com>

ivotron committed:
rptest: add two more excluded tests to quick suite

Signed-off-by: Ivo Jimenez <ivo@redpanda.com>

ivotron commented:
a good run is in https://buildkite.com/redpanda/vtools/builds/25360 . force-merging this now
